turbo.json

```diff
@@ -75,7 +75,7 @@
             "inputs": ["babel.config.json", "src/**", "test/**"]
         },
         "test:prettier": {
-            "inputs": ["src/**", "test/**"]
+            "inputs": ["*"]
```

Comment:
We operate over way more that what's in `src/` and `test/` which is probably why things sometimes slip through.